avaframe/com1DFA/com1DFA.py

```diff
@@ -163,6 +154,8 @@ def runSamos(cfg, avaDir):
     resDir = os.path.join(avaDir, 'Work', 'com1DFA')
     # Get path of module
     modPath = os.path.dirname(__file__)
+    # Standard values for parameters that can be varied
+    defValues = {'Mu' : '0.155', 'RelTh' : '1.000'}
```

Comment:
To avoid any problems in future I strongly recommend taking any of these values out to .ini files. Otherwise we run into the same problem we have right now with the legacy code where parameters are set all over the place...

Reply:
I moved these to the com1DFACfg.ini file in the new section [DEFVALUES]